Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,66 @@ namespace BootCamp.Chapter
 {
     class Program
     {
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Enter the data of the prisoner. \n");
+
+            {
```

Comment:
nit: You don't need to wrap this code in '{' with no corresponding statement. It's best to avoid unnecessary indentation.

Reply:
okay